client/src/components/viewers/FileControlsInput.vue

```diff
@@ -0,0 +1,146 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <span
+    v-if="!editing"
+    class="file-controls-input text-only"
+    @click="onTextClick"
+  >
+    {{ modelValue }}
+    {{ props.suffix }}
```

Comment:
Missing the `$msTranslate()`. And we said that we would avoid mentioning `props` in templates.

Reply:
done